internal/blueprint/blueprint.go

```diff
@@ -0,0 +1,28 @@
+// Package blueprint contains primites for representing weldr blueprints and
+// translating them to OSBuild pipelines
+package blueprint
+
+import "osbuild-composer/internal/pipeline"
+
+// A Blueprint is a high-level description of an image.
+type Blueprint struct {
+	Name        string    `json:"name"`
+	Description string    `json:"description"`
+	Version     string    `json:"version,omitempty"`
+	Packages    []Package `json:"packages"`
+	Modules     []Package `json:"modules"`
+	Groups      []Package `json:"groups"`
```

Comment:
Maybe we should only have one thing here? Weldr has all three, but puts packages in each of them. We'd need to support this still in the API, but could combine them here.

Reply:
Hm, let's address that in a follow-up? (I would have to look into how it works, this currently just preserves the behaviour).

Reply:
Yeah, I was just thinking out loud.